.github/workflows/delegator_lowperms-generic_slsa3.yml

```diff
@@ -176,7 +176,7 @@ jobs:
       - name: Checkout the project repository
         uses: slsa-framework/slsa-github-generator/.github/actions/secure-project-checkout@main
         with:
-          fetch-depth: ${{ toJson(needs.verify-token.outputs.slsa-verified-token).source.checkout.fetch_depth }}
```

Comment:
good catch! was this not working before then? But we have some scheduled test...

Reply:
in this workflow https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/e2e.verify-token.reusable.yml#L55 which is verified in https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/e2e.verify-token.reusable.yml#L54

same for `slsa-checkout-sha1`

Reply:
Yeah, I suppose this passes because the input was basically ignored and it was checked out with the default fetch-depth value. We don't seem to actually verify that the checkout happened at the proper depth in a wrapped builder action or anything as part of the tests.